### ISSUE TITLE
Add a map showing where every garden is

### ISSUE THREAD

**Issue by soobinrho** [Augie-Environmental-Studies/gardens#11]
One option is to use Google Maps API. Maybe add paces to Google?

Another option is to use an open-source mapping library.

**Comment by deepakkrish212** [comment]
hey, this problem can be fixed by leaflet js. I would be happy to take on this issue

**Comment by soobinrho** [comment]
> hey, this problem can be fixed by leaflet js. I would be happy to take on this issue

That will be great 👍

**Comment by soobinrho** [comment]
@deepakkrish212 

- Would you like to implement the maps here? https://augie-environmental-studies.github.io/gardens/docs/about/location/

- Plus, Dr. O'Hara already marked the locations for each garden in one of his articles: https://arcg.is/941Cm

**Comment by deepakkrish212** [comment]
Yeah, I think that would be an ideal place to implement it. I will get on that as soon as I figure out how to park exact places in google maps embed API
